### PR TITLE
New package: pgcli-2.1.0

### PR DESCRIPTION
**Pull request by AluisioASG**
Currently I manually patch out python-mock with unittests.mock on Python 3; might be worth packaging python3-mock instead.

### CODE REVIEW

File: srcpkgs/pgcli/template [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+short_desc="PostgreSQL CLI with autocompletion and syntax highlighting"
+maintainer="Aluísio Augusto Silva Gonçalves <aluisio@aasg.name>"
+license="BSD-3-Clause"
+homepage="https://www.pgcli.com"
+distfiles="${PYPI_SITE}/p/pgcli/pgcli-${version}.tar.gz"
+checksum=3791a3c7734802fd14fefe9d18082a95687a02631f3287bd154cd27b4f12c05c
+
+do_check() {
+	# pgcli tests require a PostgreSQL server; skip them.
+	return 0
```
Comment (maxice8) [inline srcpkgs/pgcli/template]: can be replaced to `:`

File: srcpkgs/python-backports.csv/template [hunk 2]
```diff
@@ -0,0 +1,26 @@
+# Template file for 'python-backports.csv'
+pkgname=python-backports.csv
+version=1.0.7
+revision=1
+archs=noarch
+wrksrc="backports.csv-${version}"
+build_style=python2-module
```
Comment (maxice8) [inline srcpkgs/python-backports.csv/template]: do we really need more python2 stuff ? it is EOL

**Comment by maxice8** [comment]
I'd rather have it add only python3 stuff

**Comment by AluisioASG** [comment]
Will remove those and also fix an updated dependency I missed.

**Comment by maxice8** [comment]
On Tue, Apr 16, 2019 at 08:16:48AM -0700, Aluísio Augusto Silva Gonçalves wrote:
> Will remove those and also fix an updated dependency I missed.
> 
> -- 
> You are receiving this because you commented.
> Reply to this email directly or view it on GitHub:
> https://github.com/void-linux/void-packages/pull/11015#issuecomment-483705953

Thanks

**Comment by AluisioASG** [comment]
Right now in the repository there are a few dependencies that are newer than what upstream has tested with (sqlparse and psycopg2). I'd say it might be better to wait until they catch up with those before merging this, but we'll likely end up in the same situation later on.

**Comment by maxice8** [comment]
If it works on your testing then i can merge it

**Comment by AluisioASG** [comment]
Works fine for me, and running the tests manually makes almost all of them pass (failing ones don't seem related to the newer deps).

**Pull request merged**